appengine/genetics/js/mouse.js

```diff
@@ -0,0 +1,140 @@
+/**
+ * Blockly Games: Genetics
+ *
+ * Copyright 2016 Google Inc.
+ * https://github.com/google/blockly-games
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * @fileoverview Creates a mouse.
+ * @author kozbial@google.com (Monica Kozbial)
+ */
+'use strict';
+
+goog.provide('Genetics.Mouse');
+
+goog.require('goog.math');
+
+
+/**
+ * Creates a mouse.
+ * @constructor
+ * @param {number} id The id to assign the mouse.
+ *  * @param {Genetics.Mouse.Sex} sex The sex of the mouse.
```

Comment:
There's an extra asterisk on this line.  Also, can sex be null as indicated by the JS doc?
And ID should be uppercase in the comment above (though not the param name).